packages/ui/tailwind-theming.md

```diff
@@ -0,0 +1,80 @@
+# Tailwind theming
+
+Supabase is migrating to use tailwind classes that use CSS properties.
+
+This is to support the concept of theming, so that:
+
+1. It is easy for the team to update the theme without changing 100s of files.
+2. We could offer custom or additional themes in future.
+
+## Classes
+
+The classes are a work in progress but some are now in use and will probably be permanent fixtures.
+
+The following are some of the most used classes.
+
+It is still a work in progress and class names are likely to change.
+
+### Text
+
+Default text
+
+`text-foreground`
+
+Light text
+
+`text-foreground-light`
+
+Lighter text
+
+`text-foreground-lighter`
+
+---
+
+### Background
+
+App background
+(considering changing this to "bg-body")
+`bg`
+
+Panels
+
+`bg-surface-100`
+
+`bg-surface-200`
+
+`bg-surface-300`
+
+Overlays, Dropdowns, Popovers
+
+`bg-overlay`
+
+Inputs, Radios, Checkboxes
+
+`bg-control`
+
```

Comment:
There's also a `bg-alternative` @MildTomato

Reply:
ah, didn't think anyone used it. i'll add it.

I didn't add them all as some of them seemed like they hadn't been used yet.